src/views/components/SharePanel/index.jsx

```diff
@@ -30,11 +41,31 @@ const SharePanel = () => {
       <Box>
         <MailchimpSubscribe
           url={mailchimpUrl}
-          render={({ subscribe, status, message }) =>
-            <SubscriptionForm subscribe={subscribe} status={status} message={message} />
+          render={({ subscribe, status, message }) =>(
+            <div>
+            <SubscriptionForm subscribe={subscribe} status={status} message={message} isSubscribe={isSubscribe} />
+            {status === "success" && nextStep()}
+            {console.log(status,step)}
+
+            </div>
+          )
           }
         />
       </Box>
+      <Button
+        onClick={skipToShare}
+        variant='outlined'
+        sx={{ margin: '10px' }}
+      >
+        SKIP
+      </Button>
+      </>
+       }
+
+      {step === STEPS.SUSCRIBE && <SharePage subcribe={STEPS.SUSCRIBE}/>}
+      {step === STEPS.SKIP && <SharePage subcribe={STEPS.SKIP}/>}
```

Comment:
Actually I realize I should have renamed `SharePanel` into `SubscribePanel`, then I think you could consider the `SharePage` just another Step in the `Start` component? It would be simpler as there's no dependency between the subscribe and share steps. What do you think?

Reply:
As you rightly pointed out, there is a dependency so ignore my above comment. I think we can still rename this component into `SubscribePanel`. Maybe the `subscribe` prop here could be renamed into `hasSubscribed` to make it very clear that this is a dependency on the previous action of the user. We could also have another state in this component `const isSubscribed = useState(false)` that is different from the `step` state to keep it clean, then we will have 2 steps: `{ SUBSCRIPTION: 1, SHARING: 2 }`, when the user click on Skip you only change the step to `STEPS.SHARING`, and if the user clicks on subscribe, after receiving the acknowledgement from mailchimp, you change the step to `STEP.SHARING` but also change `isSubscribed` to `true`, and this is what you pass to `hasSubscribed`.

What do you think? This way we could separate the step from the subscription status of the user.

Another option would be to simply replace the above 2 lines to: `{step !== STEPS.NOT_SUSCRIBE && <SharePage hasSubcribed={step === STEPS.SUSCRIBE}/>}`, up to you :)